compose/install/index.md

```diff
@@ -35,3 +49,7 @@ Are you looking to:
 - [Compose file reference](../compose-file/index.md)
 - [Sample apps with Compose](../samples-for-compose.md)
 
+## Feedback
+
+We have updated this page and would like to hear from you.
```

Comment:
How about "We'd love to hear from you about the improvements we've recently made to the Docker Compose installation instructions. Let us know what you think by completing this short [survey](<link>)."

CC @hyu

Reply:
Good phrasing! I approve.